modules/update-preparing-evaluate-apirequestcount-workloads.adoc

```diff
@@ -44,3 +44,4 @@ VERBS  USERNAME
 watch  system:serviceaccount:openshift-operators:3scale-operator                      manager/v0.0.0
 watch  system:serviceaccount:openshift-operators:datadog-operator-controller-manager  manager/v0.0.0
 ----
+// TODO: This example output needs to be updated; didn't get any in testing
```

Comment:
@sanchezl Can you help with what example output might look like for this for 4.13? I used flowschemas.v1beta1.flowcontrol.apiserver.k8s.io in the above examples, but I didn't get any output when I run this command with it on my test cluster. I also tried with `horizontalpodautoscalers.v2beta2.autoscaling` and also didn't get any output.

Reply:
@sanchezl Can you help with this please?

Reply:
Perfect, ty!